docs/tutorials/testing_with_control_center.rst

```diff
@@ -108,98 +101,111 @@ though without the colored highlighting.
 There are 5 major sections of the Control Center interface, which have been highlighted
 with colored boxes in the figure above and are described below:
 
```

Comment:
not sure if there's much we can do about it, but it's bothering me that the color of the text does not match the color of the image.